stl/inc/future

```diff
@@ -184,23 +184,39 @@ struct _State_deleter : _Deleter_base<_Ty> { // manage allocator and deletion st
     _Alloc _My_alloc;
 };
 
+template <class _Ty>
+union _Result_holder {
+    _Result_holder() {}
+    ~_Result_holder() {}
+
+    _Ty _Held_value;
+    operator _Ty&() noexcept {

```

Comment:
Am I missing something, or is this conversion operator unused? If not, can we remove it as extraneous?